packages/clerk-js/src/ui/components/SessionTasks/tasks/TaskChooseOrganization/ChooseOrganizationScreen.tsx

```diff
@@ -119,7 +121,10 @@ const MembershipPreview = (props: { organization: OrganizationResource }) => {
       try {
         await setActive({
           organization,
-          redirectUrl: redirectUrlComplete,
+          navigate: async () => {
```

Comment:
This is a small fix - atm the `warnMissingPendingTaskHandlers` utility is logging a warning when `TaskChooseOrganization` executes `setActive` with a pending session, but without using `navigate`